tests/actions/replicate_test.py

```diff
@@ -0,0 +1,187 @@
+import pytest
+
+from cobbler.actions import replicate
+
+
+@pytest.fixture(scope="function")
+def replicate_obj(cobbler_api):
+    return replicate.Replicate(cobbler_api)
+
+
+def test_object_creation(cobbler_api):
+    # Arrange & Act
+    result = replicate.Replicate(cobbler_api)
+
+    # Assert
+    assert isinstance(result, replicate.Replicate)
```

Comment:
Nitpick: Isn't this test redundant given the fixture that does the same (ensuring an object can be created)?

Reply:
Yes I will remove the test. In case the object creation fails we get a test setup failure which is not that nice but I agree that this is too much redundancy.